lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -56,7 +79,31 @@ def specific_matcher(arg)
           end
 
           def acceptable_pattern?(arg)
-            arg.match?(/\[.+=\w+\]/) || arg.match?(/[ >,+]/)
+            arg.match?(/[ >,+]/)
+          end
+
+          def specific_matcher_option?(arg, matcher)
+            # If `button[foo-bar_baz=foo]`, extract the "foo-bar_baz".
+            attribute = arg.slice(/\[[\w\W]*\]/)
```

Comment:
Two questions:
1. Can there be more than one attribute within the brackets?
2. Would a regular expression (with capture) be easier to use here?

Reply:
> 1. Can there be more than one attribute within the brackets?

The following code will result in a syntax error.
```ruby
expect(page).to have_css("a[href="http://example.com" disabled]")
=> Failure/Error: expect(page).to have_css('a[href="http://example.com" disabled]')
     
     Nokogiri::CSS::SyntaxError:
       unexpected ' ' after '#<Nokogiri::CSS::Node:0x00007fddf305d998>'
     # (eval):3:in `_racc_do_parse_c'
     # (eval):3:in `do_parse'
     # ./spec/capybara/sandbox_spec.rb:8:in `block (2 levels) in <top (required)>'
```

However, multiple attributes can be specified as shown below, so I've added tests for multiple attributes.
```ruby
expect(page).to have_css('a[href="http://example.com"][disabled]')
```

Reply:
>2. Would a regular expression (with capture) be easier to use here?

Indeed it is. Thank you so much. I updated it.